packages/@react-aria/textfield/src/useFormattedTextField.ts

```diff
@@ -77,6 +77,12 @@ export function useFormattedTextField(props: AriaTextFieldProps, state: Formatte
             ? input.value.slice(0, input.selectionStart - 1) + input.value.slice(input.selectionStart)
             : input.value.slice(0, input.selectionStart) + input.value.slice(input.selectionEnd);
           break;
+        case 'deleteSoftLineBackward':
+          nextValue = input.value.slice(input.selectionStart);
+          break;
+        case 'deleteHardLineBackward':
+          nextValue = input.value.slice(input.selectionStart);
+          break;
```

Comment:
We should be able to simplify this to
```suggestion
        case 'deleteSoftLineBackward':
        case 'deleteHardLineBackward':
          nextValue = input.value.slice(input.selectionStart);
          break;
```

Reply:
Thanks for catching that.

And it's working for me in Safari. Also not having the partial currency issue you mentioned. Looking into it now though.

Reply:
o, it hasn't built... i see the problem, 1 second i'll fix it